lib/archethic_web/controllers/faucet_controller.ex

```diff
@@ -0,0 +1,85 @@
+defmodule ArchEthicWeb.FaucetController do
+  @moduledoc false
+  @pool_seed Application.get_env(:archethic, __MODULE__)[:seed]
+
+  alias ArchEthic.TransactionChain.{
+    Transaction,
+    TransactionData,
+    TransactionData.Ledger,
+    TransactionData.UCOLedger
+  }
+
+  alias ArchEthic.Crypto
+
+  use ArchEthicWeb, :controller
+
+  def index(conn, __params) do
+    conn
+    |> put_resp_header("cache-control", "no-cache, no-store, must-revalidate")
+    |> put_resp_header("pragma", "no-cache")
+    |> put_resp_header("expires", "0")
+    |> render("index.html", address: "")
+  end
+
+  def create_transfer(conn, %{"address" => address}) do
+    with {:ok, address} <- Base.decode16(address, case: :mixed),
+         true <- Crypto.valid_hash?(address),
+         :ok <- transfer(address) do
+      conn
+      |> put_flash(:info, "Transferred successfully")
+      |> redirect(to: "/faucet")
```

Comment:
I didn't catch this one before, but if you can use routes helper to make the redirection dynamically generated by phoenix pointing to the `index/2` function, it will be great.